guides/deployments/postgres.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Connect an External PostgreSQL"
+description: "Learn how connect Coder to an external postgreSQL database."
+---
+
+This guide walks through the steps to deploying Coder with an external
+PostgreSQL database connected.
+
+## Connecting to an external PostgreSQL
+
+By default, Coder will deploy a [TimescaleDB](https://www.timescale.com) inside
+the Kubernetes cluster used to install Coder. However, this is _only_
+recommended for evaluation purposes, as the database is not backed up, and can
+be lost if the cluster goes down.
+
+For those reasons, we recommend bringing in your own PostgreSQL for production
+deployments, and hosting it external to the cluster. Below are the steps to do
+so:
+
+1. Spin up a PostgreSQL instance if you have not already done so
+
+1. Configure a private IP address for the PostgreSQL instance
+
+- This private IP will be referenced in the Coder helm chart configuration
+
+1. If your PostgreSQL instance has a password, follow the below:
+
+- Open the terminal, connect to your cluster and create a secret:
+
+  ```console
+  kubectl create secret generic <YOUR-DB-SECRET> \
+      --from-literal=password=<YOUR-DATABASE-PASSWORD> \
+  ```
```

Comment:
This is fine but we should encourage people to create passwords in a way where the secret isn't recorded in your shell history. For example this is a more security-conscious way to create a secret on the cmd line 
```bash 
kubectl create secret generic <NAME> --from-file=test=/dev/stdin
```

Reply:
I can update the sample command in [Step 4](https://coder.com/docs/setup/installation#installing-coder).